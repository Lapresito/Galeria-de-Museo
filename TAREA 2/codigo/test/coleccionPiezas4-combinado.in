crearColeccionPiezas
crearFecha 1/1/2001
crearPieza 1 LaxaEspadaTriste Sapphire Frost
agregarColeccionPiezas
crearFecha 3/2/2004
crearPieza 3 SacraLataFurtiva River Rain
agregarColeccionPiezas
crearFecha 1/1/2005
crearPieza 5 GozadaEspadaDesquiciada River Rain
agregarColeccionPiezas
crearFecha 5/4/2006
crearPieza 7 RaraMangueraDesquiciada River Rain
agregarColeccionPiezas
crearFecha 5/4/2007
crearPieza 4 MalditaLataFunebre River Rain
agregarColeccionPiezas
crearFecha 5/4/2008
crearPieza 6 AltaBarcaAfilada River Rain
agregarColeccionPiezas
removerDeColeccionPiezas 5
removerDeColeccionPiezas 1
esVaciaColeccionPiezas
crearFecha 1/1/2005
crearPieza 5 GozadaEspadaDesquiciada River Rain
agregarColeccionPiezas
obtenerPiezaColeccionPiezas 5
obtenerPiezaColeccionPiezas 4
obtenerPiezaColeccionPiezas 6
esVaciaColeccionPiezas
imprimirColeccionPiezas
liberarColeccionPiezas
Fin
