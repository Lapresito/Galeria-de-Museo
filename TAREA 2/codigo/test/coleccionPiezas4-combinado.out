Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> Coleccion de piezas creada de forma exitosa.
2> Fecha creada en forma exitosa.
3> Pieza creada en forma exitosa.
4> Pieza agregada de forma exitosa.
5> Fecha creada en forma exitosa.
6> Pieza creada en forma exitosa.
7> Pieza agregada de forma exitosa.
8> Fecha creada en forma exitosa.
9> Pieza creada en forma exitosa.
10> Pieza agregada de forma exitosa.
11> Fecha creada en forma exitosa.
12> Pieza creada en forma exitosa.
13> Pieza agregada de forma exitosa.
14> Fecha creada en forma exitosa.
15> Pieza creada en forma exitosa.
16> Pieza agregada de forma exitosa.
17> Fecha creada en forma exitosa.
18> Pieza creada en forma exitosa.
19> Pieza agregada de forma exitosa.
20> Pieza con id 5 removida con exito.
21> Pieza con id 1 removida con exito.
22> La coleccion de piezas NO es vacia.
23> Fecha creada en forma exitosa.
24> Pieza creada en forma exitosa.
25> Pieza agregada de forma exitosa.
26> Pieza: 5
GozadaEspadaDesquiciada-River Rain
Creada en: 1/1/2005
27> Pieza: 4
MalditaLataFunebre-River Rain
Creada en: 5/4/2007
28> Pieza: 6
AltaBarcaAfilada-River Rain
Creada en: 5/4/2008
29> La coleccion de piezas NO es vacia.
30> Pieza: 3
SacraLataFurtiva-River Rain
Creada en: 3/2/2004
Pieza: 4
MalditaLataFunebre-River Rain
Creada en: 5/4/2007
Pieza: 5
GozadaEspadaDesquiciada-River Rain
Creada en: 1/1/2005
Pieza: 6
AltaBarcaAfilada-River Rain
Creada en: 5/4/2008
Pieza: 7
RaraMangueraDesquiciada-River Rain
Creada en: 5/4/2006
31> Coleccion liberada con exito.
32> Fin.
